### ISSUE TITLE
Add button to blog and update sonification code 

### ISSUE THREAD

**Issue by dc3347** [Barnard-PL-Labs/tsltools#17]
03/07/2022
Update: This week I worked with Rhea to update the [tonejs sonification](https://barnard-pl-labs.github.io/tsltools/tonejs/) code to better reflect the generated code from the TSL specification. The code works on initial click, however at the moment there is a slight play/pause issue upon clicking play after clicking pause. Commit: [57b2d2e](https://github.com/Barnard-PL-Labs/tsltools/commit/57b2d2e5409bf12a8b32b6fa00c7b189b5daa653). 

We were also able to add the the play/pause button to the 4 examples in the [blog post](https://barnard-pl-labs.github.io/tsltools/tsl_intro_blogpost.html), but had to do it in an html file. Currently we are aware that nothing plays on the blog page when all 4 buttons are added (1 button works fine though, however. See commit below). We think this is due to the fact that the 4 button ids are the identical. We are working to resolve this before the meeting tomorrow. Commit: [94fb63a](https://github.com/Barnard-PL-Labs/tsltools/commit/94fb63aad967dc35e50e00116986c064148610e3).

**Comment by Rheakk** [comment]
Update (3/7/22): Danielle and I modified our [sonification of two notes](https://barnard-pl-labs.github.io/tsltools/tonejs/) to more closely match our TSL specification. We did this by separating out the actual looping / playing of notes from the way the state machine for our notes is updated (the former could just be dumped in when synthesizing code for musical purposes, and the latter will rely on our TSL specification). As Danielle stated in her update - the two notes correctly alternate when the button in our html page is clicked for the first time, but if we press the button to pause and press it again to play, the two notes play simultaneously. We are still figuring out why this is working. Commit: [57b2d2e](https://github.com/Barnard-PL-Labs/tsltools/commit/57b2d2e5409bf12a8b32b6fa00c7b189b5daa653).
